packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -408,6 +408,65 @@ const defaultConfigs = {
       }
     ]
   },
+  "USD/bBadger": {
+    type: "expression",
+    // Note: lower-case variables are intermediate, upper-case are configured feeds.
+    expression: `
+      wbtc_usd = mean(WBTC_ETH_SUSHI, WBTC_ETH_UNI) / USDETH;
+      badger_usd_sushi = wbtc_usd * BADGER_WBTC_SUSHI;
+      badger_usd_uni = wbtc_usd * BADGER_WBTC_UNI;
+      badger_usd = median(badger_usd_sushi, badger_usd_uni, BADGER_USD_HUOBI);
+      1 / (badger_usd * BBADGER_BADGER)
+    `,
+    lookback: 7200,
+    minTimeBetweenUpdates: 60,
+    twapLength: 7200,
```

Comment:
How does `twapLength` work as part of the expression price feed? Does it only apply to the uniswap feeds in `customFeeds`?